src/main/webapp/app/assessment/assessment-header/assessment-header.component.html

```diff
@@ -73,6 +73,24 @@ <h3 class="top-container">
         >
             You have the lock for this assessment
         </span>
+        <!-- Highlight the difference between first and second correction -->
+        <button
+            *ngIf="!highlightDifferences && result && correctionRound > 0"
+            class="btn ml-2 btn-primary"
+            (click)="toggleHighlightDifferences()"
+            [disabled]="saveBusy || submitBusy || cancelBusy"
+        >
+            <span jhiTranslate="artemisApp.assessment.differenceActivate">Activate Diff-View</span>
+        </button>
+        <button
+            *ngIf="highlightDifferences && result && correctionRound > 0"
+            class="btn ml-2 btn-primary"
+            (click)="toggleHighlightDifferences()"
+            [disabled]="saveBusy || submitBusy || cancelBusy"
+        >
+            <span jhiTranslate="artemisApp.assessment.differenceDeactivate">Deactivate Diff-View</span>
+        </button>
```

Comment:
I think you can just change the button text with highlightDifferences

Reply:
yes that's a good catch, I added that